test/functional/p2p_disconnect_ban.py

```diff
@@ -4,6 +4,7 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Test node disconnect and ban behavior"""
 import time
+import os
```

Comment:
Would be nice to use the Path interface for new code, given that `chain_path` is already Path?

https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module

Reply:
Sounds good, gonna address it!